### ISSUE TITLE
Path issues in data augmentation notebook in Windows and OSX

### ISSUE THREAD

**Issue by iyerkrithika21** [SCIInstitute/ShapeWorks#1293]
Paths in `DataAugmentationUtils` have to change to handle Windows path.
Right now, the notebook fails with the following error:
```
Running point based data augmentation.
The PCA modes of particles being retained :  5
Variablity preserved: 0.9586381419830339
Fitting Gaussian distribution...
Generating 1/50

---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-18-c2a03822e17d> in <module>
      2 output_directory = '..\\Output\\GaussianAugmentation\\'
      3 sampler_type = "gaussian"
----> 4 embedded_dim = DataAugmentationUtils.runDataAugmentation(output_directory, img_list, local_particle_list, num_samples, num_dim, percent_variability, sampler_type)
      5 aug_data_csv = output_directory + "\\TotalData.csv"

~\miniconda3\envs\shapeworks\lib\site-packages\DataAugmentationUtils\__init__.py in runDataAugmentation(out_dir, img_list, local_point_list, num_samples, num_dim, percent_variability, sampler_type, mixture_num, processes, world_point_list)
     16 def runDataAugmentation(out_dir, img_list, local_point_list, num_samples=3, num_dim=0, percent_variability=0.95, sampler_type="KDE", mixture_num=0, processes=1, world_point_list=None):
     17     print("Running point based data augmentation.")
---> 18     num_dim = DataAugmentation.point_based_aug(out_dir, img_list, local_point_list, num_samples, num_dim, percent_variability, sampler_type, mixture_num, processes, world_point_list)
     19     print("Done.")
     20     return num_dim

~\miniconda3\envs\shapeworks\lib\site-packages\DataAugmentationUtils\DataAugmentation.py in point_based_aug(out_dir, orig_img_list, orig_point_list, num_samples, num_dim, percent_variability, sampler_type, mixture_num, processes, orig_world_point_list)
     84                 base_particles_path = orig_point_list[base_index]
     85                 if processes==1:
---> 86                         gen_image_path = Utils.generate_image(out_dir, gen_points_path, base_image_path, base_particles_path)
     87                         gen_image_paths.append(gen_image_path)
     88                 else:

~\miniconda3\envs\shapeworks\lib\site-packages\DataAugmentationUtils\Utils.py in generate_image(out_dir, gen_particles, base_image, base_particles)
     52                 "warp-image", "--source", base_particles, "--target", gen_particles, "--stride", "2",
     53 		"write-image", "--name", gen_image]
---> 54         subprocess.check_call(cmd)
     55         return gen_image
     56 

~\miniconda3\envs\shapeworks\lib\subprocess.py in check_call(*popenargs, **kwargs)
    356     check_call(["ls", "-l"])
    357     """
--> 358     retcode = call(*popenargs, **kwargs)
    359     if retcode:
    360         cmd = kwargs.get("args")

~\miniconda3\envs\shapeworks\lib\subprocess.py in call(timeout, *popenargs, **kwargs)
    337     retcode = call(["ls", "-l"])
    338     """
--> 339     with Popen(*popenargs, **kwargs) as p:
    340         try:
    341             return p.wait(timeout=timeout)

~\miniconda3\envs\shapeworks\lib\subprocess.py in __init__(self, args, bufsize, executable, stdin, stdout, stderr, preexec_fn, close_fds, shell, cwd, env, universal_newlines, startupinfo, creationflags, restore_signals, start_new_session, pass_fds, encoding, errors, text)
    798                                 c2pread, c2pwrite,
    799                                 errread, errwrite,
--> 800                                 restore_signals, start_new_session)
    801         except:
    802             # Cleanup if the child failed starting.

~\miniconda3\envs\shapeworks\lib\subprocess.py in _execute_child(self, args, executable, preexec_fn, close_fds, pass_fds, cwd, env, startupinfo, creationflags, shell, p2cread, p2cwrite, c2pread, c2pwrite, errread, errwrite, unused_restore_signals, unused_start_new_session)
   1205                                          env,
   1206                                          os.fspath(cwd) if cwd is not None else None,
-> 1207                                          startupinfo)
   1208             finally:
   1209                 # Child is launched. Close the parent's copy of those pipe

FileNotFoundError: [WinError 2] The system cannot find the file specified

```

**Comment by jadie1** [comment]
@iyerkrithika21 and @archanasri Can you try it on the data_aug branch? You will need to run pip again:
`pip install ShapeWorks/Python/DataAugmentationUtilsPackage/`

**Comment by iyerkrithika21** [comment]
Same error with the changes.
For the command 


`['shapeworks', 'read-image', '--name', '../../Data/femur-v0/groomed/images/n12_L_1x_hip.isores.pad.com.center.aligned.cropped.nrrd', 'warp-image', '--source', '../../Data/femur-v0/shape_models/femur/1024/n12_L_femur.isores.pad.com.center.aligned.clipped.cropped.tpSmoothDT_local.particles', '--target', '../Output/GaussianAugmentation/Generated-Particles/Generated_sample_0001.particles', '--stride', '2', 'write-image', '--name', '../Output/GaussianAugmentation/Generated-Images/Generated_sample_0001.nrrd']`
It is creating the file `Generated_sample_0001.particles`, but fails at the `write-image` part

**Comment by jadie1** [comment]
Fixed in PR #1307
